builder.md

```diff
@@ -5,43 +5,44 @@
 "SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL" in this document are to be
 interpreted as described in [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119).*
 
-A base16 builder is a tool that builds application specific themeing configurations. It does this by using base16 scheme files (containing a collection of colors) and base16 template files (instructions concerning how to build the application specific files).
+This document describes the requirments and basic functionality of theme builders. Builders are generally designed for template maintainers' ease of use. Template maintainers SHOULD provide built versions of their template so the end user doesn't need to be aware of builders.
 
-Builders are generally designed for template maintainers' ease of use. Template maintainers SHOULD provide built versions of their template so the end user doesn't need to be aware of the builder.
+## Glossary
 
-## Definitions
+A _scheme_ (or color scheme) is a palette of colors along with some metadata, generally stored in a YAML file. For example: the [base16 solarized-dark scheme](https://github.com/tinted-theming/base16-schemes/blob/main/solarized-dark.yaml)
 
-A _base16 scheme_ is a YAML file that represents a palette of 16 colors. For
-example: the [solarized
-scheme](https://github.com/tinted-theming/base16-schemes/blob/main/solarized-dark.yaml)
+A _scheme system_ generally consists of a styling guide and schemes. For example the base16 [styling guide](./styling.md) and [schemes](https://github.com/tinted-theming/base16-schemes).
 
-A _base16 template_ is a mustache file that acts as a blueprint; it represents
-how to translate the scheme into an application's desired format. For example: the [vim
-template](https://github.com/tinted-theming/base16-vim/blob/main/templates/default.mustache)
-is used to convert a _base16 scheme_ into a vim colorscheme.
+A _template_ is a mustache file which acts as a blueprint; it represents how to translate the scheme into an application's desired format. For example: the [base16-vim template](https://github.com/tinted-theming/base16-vim/blob/main/templates/default.mustache) is used to convert a _base16 scheme_ into a vim colorscheme.
```

Comment:
I know this was here already, but `colorscheme` is used, while `color scheme` is mentioned earlier. I tend to use `colorscheme` when I refer to the generated "theme" files from the templates and `scheme` as the schemes aka tinted-theming/base16-schemes, not sure where I picked that up from.

Should we remove this `colorscheme` mention to reduce confusion with `color scheme` which you've mentioned is synonymous with `scheme` in this file? If so, what do we call the generates files? `themes`? Or maybe we don't need to mention that in this document? Not sure, just raising the topic.

Reply:
Good callout - I swapped this to Emacs which calls them themes rather than colorschemes like vim.